### DISCUSSION TITLE
EPELにあるパッケージのインストールについて

### DISCUSSION THREAD

**Discussion by WTB80** [nig-sc/nigsc_homepage2#297] [Q&A] [answered]
SeuratでSingle cell RNA-sequence解析を行いたいと思っております。
RでHDF5ファイルを読み込むには、"hdf5r"ライブラリをインストールする必要があるのですが、
それに先立って"HDF5(hdf5-devel)"パッケージをインストール必要があります。
"HDF5"パッケージはExtra Package for Enterprise Linux(EPEL)にあります。
EPELにあるパッケージはRed Hat Enterprise Linux(RHEL)のサポート対象外になるとのことですが、
EPELにあるパッケージをインストールしても構いませんでしょうか。

**Comment by sc-fs** [comment]
ユーザのホーム環境下へのインストールには、制約はありません。
必要であれば、conda,spackをご利用下さい。

Seuratのインストールは以下が参考になるかと思います。
https://qiita.com/amufaamo/items/c185e7ff88c90b443946

**Reply by WTB80** [reply]
ご回答ありがとうございます。
ルート権限がないため、下記のコマンドで入ることができませんでした。
yum install epel-release
どのようにすれば良いか教えて頂けましたら幸いです。
宜しくお願い致します。

**Reply by sc-fs** [reply, marked as answer]
残念ながら、ルート権限が必要な操作は、許可していません。

R環境へのhdf5rインストールでつまずいているのであれば、
先のページあるように、conda環境上にて対応するのが良いと思います。
なお、R環境にてinstall.packages("hdf5r")のインストール時にlibhdf5_hl.so.100がみつからないエラーが発生した場合は、
環境変数のLD_LIBRARY_PATHにlibhdf5_hl.so.100の場所を追加してください。
`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/anaconda3/lib/ `

他の方法としては、HDF5関連のソースコードを入手して、自身でバイナリを作成する方法となります。
